core/arch/arm/kernel/cache_helpers_a64.S

```diff
@@ -102,6 +102,9 @@ LOCAL_FUNC do_dcsw_op , :
 	cbz	x3, exit
 	adr	x14, dcsw_loop_table	// compute inner loop address
 	add	x14, x14, x0, lsl #5	// inner loop is 8x32-bit instructions
+#if defined(CFG_CORE_BTI)
```

Comment:
Could this be done:
```
#if defined(CFG_CORE_BTI)
xxx	// inner loop is 8x32-bit + "bti j" instructions
#else
add	x14, x14, x0, lsl #5	// inner loop is 8x32-bit instructions
#endif
```

Reply:
I tried to keep it same as they had done in TF-A to maintain consistency . https://github.com/ARM-software/arm-trusted-firmware/blob/683bb4d7bdfd42e6e026902c43797f132b2a75d5/lib/aarch64/cache_helpers.S#L94

but if you want me to change it, I can try doing that

Reply:
No, let's keep it as it is.